enable_location.sh

```diff
@@ -25,8 +25,13 @@ ERROR_REDIRECT_CODES=$(get_id_var ${LOCATION_ID} ERROR_REDIRECT_CODES)
 ENABLE_WEB_SOCKETS=$(get_id_var ${LOCATION_ID} ENABLE_WEB_SOCKETS)
 ADD_NGINX_LOCATION_CFG=$(get_id_var ${LOCATION_ID} ADD_NGINX_LOCATION_CFG)
 
+# Backwards compatability
+if [ "`echo ${PROXY_SERVICE_HOST} | grep -i 'https\?://'`" = "" ]; then
```

Comment:
I like it transparent and default to http if not specified. We can always worry about other not http proxying in a separate PR...